addons/circulation/functions/fnc_cprLocal.sqf

```diff
@@ -101,7 +101,7 @@ switch (_reviveObject) do {
 };
 
 if (_reviveObject isEqualTo "AED" || _reviveObject isEqualTo "AEDX" || _reviveObject isEqualTo "AEDStation" || _reviveObject isEqualTo "AEDXVehicle") exitWith {
-    _chance = _chance + (_amiBoost + _lidoBoost * _epiBoost);
+    _chance = _chance + (_amiBoost + _lidoBoost + _epiBoost);
```

Comment:
Instead of this, we could do it like
_chance = _chance + (_amiBoost + (1 + _lidoBoost) * epiBoost); 🤔